toonz/sources/toonz/menubar.cpp

```diff
@@ -1414,6 +1414,7 @@ QMenuBar *StackedMenuBar::createFullMenuBar() {
   windowsMenu->addSeparator();
   addMenuItem(windowsMenu, MI_OpenFileBrowser);
   addMenuItem(windowsMenu, MI_OpenFileBrowser2);
+  addMenuItem(windowsMenu, MI_OpenAdvancedColorSelector);
```

Comment:
If you have to make another update to this PR, can you move this command's menu location up, below one of these lines:

>1403  addMenuItem(windowsMenu, MI_OpenColorModel);
or
>1400  addMenuItem(windowsMenu, MI_OpenStyleControl);

It would make more sense under that grouping of commands.

Please also add the command to `stuff\profiles\layouts\rooms\Default\menubar_template.xml` in the same position.